### ISSUE TITLE
Improve Ecopedia integration. User feedback

### ISSUE THREAD

**Issue by johnkslg** [StrangeLoopGames/EcoIssues#19486]
From our programmer player:

"Clicking the [?] button for Court item tooltip just brings me to this
![image](https://user-images.githubusercontent.com/3536496/100292824-7144b600-2f36-11eb-81cf-44814ee6adf3.png)

Which is absolutely pointless, that's the tooltip I just clicked from. And you just thought me to ignore that button.

“Civic action component” and “civic object component” seem like a very programmer way to explain something.
Just tell me what objects do, “Civic action component” is not something tangible in the game I should know about, they are just tabs in a gui.
The information in “Civic action component” and “civic object component” also seems to be rather redundant or split with the information in the "Government” section."

**Comment by johnkslg** [comment]
He's got a good point, lets make the buttons for all world objects take them to the help for the most-relevant component.

**Comment by SlayksWood** [comment]
@johnkslg  I disagree with this. There are many options to open Components in Ecopedia and only one to open the table.
1. First of all you can click here:
![image](https://user-images.githubusercontent.com/45708377/100316820-515fd180-2fcc-11eb-8a73-0db2fe626a55.png)
and open any Component which you want. Not only one.
2. After opening 'useless' Court, Players can click here to bring them to any component:
![image](https://user-images.githubusercontent.com/45708377/100317264-2033d100-2fcd-11eb-9471-129acc7bd0f7.png)
3. You can click component just in tooltip to open in ecopedia:
![image](https://user-images.githubusercontent.com/45708377/100317071-c59a7500-2fcc-11eb-8b10-5cc0966cc946.png)
4. 3rd point is not very clear and the only one change what I'd like to make - add ecopedia button [?] to this tooltip too:
![image](https://user-images.githubusercontent.com/45708377/100317512-86b8ef00-2fcd-11eb-9b8b-77c9e592c88a.png)

In addition, we have a couple of tables (Government Office, Capitol) where there are 2 important components.
